### PR TITLE
Adding tpv variable for cooler_cload_tabix tool

### PR DESCRIPTION
**Pull request by SaimMomin12**

### CODE REVIEW

**Comment by bgruening** [comment]
@sanjaysrikakulam ORG has this change in the default tool. https://github.com/galaxyproject/usegalaxy-playbook/blob/08336fc92f658dd34714f6d8408d02c56502c905/env/common/templates/galaxy/config/tpv/defaults.yaml.j2#L28

I think we should do similar... What do you think.
We need this pr urgently for a collaboration and a failing tool. But a general solution would be nice

**Comment by bgruening** [comment]
See line 390/395 @SaimMomin12

**Comment by SaimMomin12** [comment]
@bgruening Should this be updated in the following way?

```
env: 
    - HDF5_USE_FILE_LOCKING: "FALSE"
```

File: files/galaxy/tpv/tool_defaults.yml [hunk 1]
```diff
@@ -8,7 +8,10 @@ tools:
     mem: cores * 3.8
     gpus: 0
     env:
-      GALAXY_MEMORY_MB: "{int(mem * 1024)}" # set 5/2023 might be moved to runner or tool wrappers, related to Galaxy issue 15952
+      - name: GALAXY_MEMORY_MB
+        value: "{int(mem * 1024)}" # set 5/2023 might be moved to runner or tool wrappers, related to Galaxy issue 15952
+      - name: HDF5_USE_FILE_LOCKING
```
Comment (bgruening) [inline files/galaxy/tpv/tool_defaults.yml:13]: maybe also take over the SINGULARITY env var?

**Comment by sanjaysrikakulam** [comment]
> @sanjaysrikakulam ORG has this change in the default tool. https://github.com/galaxyproject/usegalaxy-playbook/blob/08336fc92f658dd34714f6d8408d02c56502c905/env/common/templates/galaxy/config/tpv/defaults.yaml.j2#L28
> 
> I think we should do similar... What do you think. We need this pr urgently for a collaboration and a failing tool. But a general solution would be nice

Yes, setting it in the tool default is fine, I think (I did not find it set to `True` anywhere in the ORG's confs). I have pushed a new commit adding the ENV `HDF5_USE_FILE_LOCKING: "False"` to tool default and removed the tool-specific setting and the tool entry since the tool does not have any exclusive requirements. Feel free to merge and deploy.

**Comment by bgruening** [comment]
@sanjaysrikakulam wants to go all in - nice. Please feel free to merge and redeploy. Can you let @SaimMomin12 know when it was restarted?

File: files/galaxy/tpv/tool_defaults.yml (hunk 1 above)
Comment (sanjaysrikakulam) [reply files/galaxy/tpv/tool_defaults.yml:13]: Sure! Will push a new commit adding SINGULARITYENV_HDF5_USE_FILE_LOCKING.

**Review by sanjaysrikakulam — APPROVED** [review]

**Comment by sanjaysrikakulam** [comment]
@SaimMomin12, the config is deployed. Please test.

**Comment by SaimMomin12** [comment]
@sanjaysrikakulam thanks for the quick patch, I will test the tool and let you know :)

**Pull request merged**